tests/addresslib/address_test.py

```diff
@@ -138,7 +138,7 @@ def test_edge_cases():
 def test_display_name__to_full_spec():
     eq_('"foo (\\"bar\\") blah" <foo@bar.com>',
         EmailAddress('foo ("bar") blah', 'foo@bar.com').full_spec())
-    eq_('"foo. bar" <foo@bar.com>',
+    eq_('foo. bar <foo@bar.com>',
```

Comment:
@horkhe Yes, although its a special character and suppose to be quoted. The obsolete syntax allows for it to appear unquoted. The obsolete syntax isn't technically suppose to be used for formatting the address, only for parsing it, but it shouldn't cause any failures downstream. If it does we'll revert.